cmd/check_container.go

```diff
@@ -201,8 +201,9 @@ var checkContainerCmd = &cobra.Command{
 
 			log.Info("Test results have been submitted to Red Hat.")
 			log.Info("These results will be reviewed by Red Hat for final certification.")
-			log.Infof("The container's image id is: %s.", certImage.ImageID)
-			log.Infof(fmt.Sprintf("Please check %s to monitior the progress.", buildConnectURL(projectId)))
+			log.Infof("The container's image id is: %s.", certImage.ID)
+			log.Infof("Please check %s to view scan results.", buildScanResultsURL(projectId, certImage.ID))
+			log.Infof(fmt.Sprintf("Please check %s to monitior the progress.", buildOverviewURL(projectId)))
```

Comment:
nit typo - "monitior" should be "monitor"

Reply:
Good catch!